lib/mpool/src/mblock_file.c

```diff
@@ -92,7 +92,7 @@ struct mblock_file {
     int            fileid;
     int            fd;
 
-    atomic_int *wlenv;
+    atomic_uint *wlenv;
```

Comment:
Would rather this be based on fixed size integers. Seems like you use uint32_t later. Maybe `atomic_uint_least32_t`

Reply:
Fixed, thnx!